src/sdsc/__init__.py

```diff
@@ -412,6 +431,8 @@ def matchcontextpattern(words, wordposition, totalwords,
     return bool(contextpattern[2]) == match
 
 def preparetermpatterns(term, useonepattern):
+    # FIXME toms (doc): missing docstring
+    # FIXME toms (style): remove global variable
```

Comment:
It was just meant as a reminder. I know it is not really easy, but I will come up with an idea in the future. Give me some more time.